openapi3filter/req_resp_decoder.go

```diff
@@ -208,8 +208,14 @@ func decodeStyledParameter(param *openapi3.Parameter, input *RequestValidationIn
 	var dec valueDecoder
 	switch param.In {
 	case openapi3.ParameterInPath:
+		if input.PathParams == nil {
```

Comment:
Could you check `len` instead of `nil`-ness here?

Reply:
the PathParams is a map type. so i think we check nil is better.